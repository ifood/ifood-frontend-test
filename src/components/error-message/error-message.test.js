import React from 'react'
import { render } from '@testing-library/react'
import { ErrorMessage } from '.'

describe('ErrorMessage', () => {
  it('should render component correctly', () => {
    const { getByText } = render(<ErrorMessage />)
    const linkElement = getByText(/ErrorMessage/i)
    expect(linkElement).toBeInTheDocument()
  })
})
