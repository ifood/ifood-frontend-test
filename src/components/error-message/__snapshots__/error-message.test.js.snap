// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage should render component correctly 1`] = `
<DocumentFragment>
  <div>
    ErrorMessage
  </div>
</DocumentFragment>
`;
